core/src/main/java/org/neo4j/ogm/session/delegates/LoadByTypeDelegate.java

```diff
@@ -48,12 +50,12 @@ public LoadByTypeDelegate(Neo4jSession session) {
     public <T> Collection<T> loadAll(Class<T> type, Filters filters, SortOrder sortOrder, Pagination pagination,
```

Comment:
Can we get a java doc here to describe the new behaviour?

Reply:
Done. Also added the changes in label determination to the changelog.